app/models/katello/package_group.rb

```diff
@@ -4,6 +4,11 @@ class PackageGroup < Katello::Model
 
     CONTENT_TYPE = "package_group".freeze
     has_many :roots, :through => :repositories, :class_name => "Katello::RootRepository"
+    has_many :content_view_filter_rules,
+             class_name: "Katello::ContentViewPackageGroupFilterRule",
+             primary_key: :pulp_id,
+             foreign_key: :uuid,
+             dependent: :destroy
```

Comment:
based on the error, i think you've got primary_key and foreign_key swapped here.